front/app/modules/commercial/idea_custom_fields/admin/containers/projects/edit/ideaform/IdeaCustomField.tsx

```diff
@@ -233,103 +126,86 @@ export default memo<Props>(
-                  {canSetEnabled && (
-                    <ToggleContainer>
-                      <StyledToggle
-                        checked={fieldEnabled}
-                        onChange={handleEnabledOnChange}
-                        label={<FormattedMessage {...messages.enabled} />}
-                        labelTextColor={colors.adminTextColor}
-                        className={`
-                        e2e-${ideaCustomField.attributes.code}-enabled-toggle-label
-                      `}
-                      />
-                      <IconTooltip
-                        content={
-                          <FormattedMessage
-                            {...messages.enabledTooltipContent}
-                          />
-                        }
-                      />
-                    </ToggleContainer>
-                  )}
-                  {fieldEnabled && canSetRequired && (
-                    <ToggleContainer>
-                      <StyledToggle
-                        checked={fieldRequired}
-                        onChange={handleRequiredOnChange}
-                        label={<FormattedMessage {...messages.required} />}
-                        labelTextColor={colors.adminTextColor}
-                        className={`
-                          e2e-${ideaCustomField.attributes.code}-required-toggle-label
-                      `}
-                      />
-                      <IconTooltip
-                        content={
-                          <FormattedMessage
-                            {...messages.requiredTooltipContent}
-                          />
-                        }
-                      />
-                    </ToggleContainer>
-                  )}
-                </Toggles>
-
-                {fieldEnabled && (
-                  <Suspense fallback={<Spinner />}>
-                    <QuillMutilocWithLocaleSwitcher
-                      id={`${ideaCustomField.id}-description`}
-                      noImages={true}
-                      noVideos={true}
-                      noAlign={true}
-                      valueMultiloc={descriptionMultiloc}
-                      onChange={handleDescriptionOnChange}
-                      label={
-                        <LocaleSwitcherLabelText>
-                          <FormattedMessage {...messages.descriptionLabel} />
-                        </LocaleSwitcherLabelText>
-                      }
-                    />
-                  </Suspense>
-                )}
-              </CollapseContainerInner>
-            </CollapseContainer>
-          </CSSTransition>
-        </Container>
+        <Accordion
+          title={
+            <Box
+              onMouseDown={removeFocusAfterMouseClick}
+              className={`
+              e2e-${ideaCustomField.attributes.code}-setting-collapsed
+            `}
+            >
+              <Title color="adminTextColor" fontSize="l" fontWeight="normal">
```

Comment:
You should normally use a variant (`h3` in this case?) for the title component and the styles are defined as part of that. Passing fontSize and fontWeight explicitly should be the exception.

Reply:
Let's go with `h3` for now. I noticed it wasn't exactly matching what was there. I agree with the direction though but if we truly want it, we need to restrict passing in those props and only allow the variant or other such props.